pkg/server/chaosd/jvm.go

```diff
@@ -60,66 +61,76 @@ const bmInstallCommand = "bminstall.sh -b -Dorg.jboss.byteman.transform.all -Dor
 const bmSubmitCommand = "bmsubmit.sh -p %d -%s %s"
 
 func (j jvmAttack) Attack(options core.AttackConfig, env Environment) (err error) {
+	// install agent
 	attack := options.(*core.JVMCommand)
-
-	if attack.Type == core.JVMInstallType {
-		return j.install(attack)
-	} else if attack.Type == core.JVMSubmitType {
-		return j.submit(attack)
-	}
-
-	return errors.Errorf("attack type %s not supported", attack.Type)
-}
-
-func (j jvmAttack) install(attack *core.JVMCommand) error {
-	var err error
-
 	bmInstallCmd := fmt.Sprintf(bmInstallCommand, attack.Port, attack.Pid)
 	cmd := exec.Command("bash", "-c", bmInstallCmd)
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		log.Error(string(output), zap.Error(err))
-		return err
+		// this error will occured when install agent more than once, and will ignore this error and continue to submit rule
+		errMsg1 := "Agent JAR loaded but agent failed to initialize"
+
+		// these two errors will occured when java version less or euqal to 1.8, and don't know why
+		// but it can install agent success even with this error, so just ignore it now.
+		// TODO: Investigate the cause of these two error
+		errMsg2 := "Provider sun.tools.attach.LinuxAttachProvider not found"
+		errMsg3 := "install java.io.IOException: Non-numeric value found"
+		if !strings.Contains(string(output), errMsg1) && !strings.Contains(string(output), errMsg2) &&
```

Comment:
Since errMsgX are all only used once here , I think giving the literal string is enough.

Reply:
It's convenient to add comments, and avoid lines that are too long

Reply:
OK.